message/validation/partial_validation.go

```diff
@@ -23,10 +23,10 @@ func (mv *messageValidator) validatePartialSignatureMessage(
 ) {
 	ssvMessage := signedSSVMessage.SSVMessage
 
-	if len(ssvMessage.Data) > maxPartialSignatureMsgSize {
+	if len(ssvMessage.Data) > genesisValidation.MaxPartialSignatureMsgSize {
 		e := ErrSSVDataTooBig
 		e.got = len(ssvMessage.Data)
-		e.want = maxPartialSignatureMsgSize
```

Comment:
we should not mix genesis and non-genesis code. this file should use the post-fork constants.